content/getting-started/setting-up.md

```diff
@@ -54,12 +54,12 @@ After the build has finished successfully, you will immediately have **artifacts
 
 ## Multiple projects in one repository
 
-Codemagic supports monorepos and is able to detect multiple Flutter projects in a repository provided that each project has its `pubspec.yaml` file.
+Codemagic supports monorepos and is able to detect multiple Flutter projects in a repository or projects not in the repository root provided that each project has its `pubspec.yaml` file with `flutter` dependency.
```

Comment:
Flutter, with big L

Reply:
I think it's better to keep it with small one, in `pubspec.yaml` it looks like this 
```
dependencies:
  flutter:
    sdk: flutter
```

I meant about the name of dependency, not Flutter itself